Source/WebCore/platform/graphics/FontDescription.cpp

```diff
@@ -70,7 +70,7 @@ FontDescription::FontDescription()
 
 static AtomString computeSpecializedChineseLocale()
 {
-    for (auto& language : userPreferredLanguages()) {
+    for (auto& language : userPreferredLanguages(ShouldMinimizeLanguages::No)) {
```

Comment:
Could there be any fingerprinting regressions from adding the argument ShouldMinimizeLanguages::No here?

Reply:
@litherum is the expert here. The thing is that if we minimized, we get several failures tests which seem to reflect actual issues our customers would experience. Those issues are regressions from us doing locale minimization a little while back but we failed to notice because the tests kept passing because our test infrastructure for locale override was a little too fake (and didn't fully test code paths the customers rely on).

The code where I stop minimizing doesn't seem to expose the locales directly to the Web (unlike `navigator.languages` which was the fingerprint vector). Seems like even if it exposed something to the Web, it would be about traditional vs simplified Chinese (or one specific Chinese locale).

Reply:
This is interesting, and concerning. Can we reset this back to `ShouldMinimizeLanguages::Yes`, mark those tests as failing, file a bug about it, and assign the bug to me?

Reply:
I will do that.